lldb/source/Plugins/Instruction/RISCV/EmulateInstructionRISCV.h

```diff
@@ -99,6 +100,8 @@ class EmulateInstructionRISCV : public EmulateInstruction {
 private:
   /// Last decoded instruction from m_opcode
   DecodeResult m_decoded;
+  /// Last tried to be decoded instruction expected size.
```

Comment:
I might call this `Last decoded instruction size`.

Reply:
Well, I tried to emphasize that even if we failed to fully decode instruction, we are still able to fill this information. I'll change to 
```
Last decoded instruction size estimate.
```